frontend/package-lock.json

```diff
@@ -1290,6 +1290,100 @@
         "glob-to-regexp": "^0.3.0"
       }
     },
+    "@nivo/axes": {
```

Comment:
There seems to be a syntax error in the `package-lock.json` file. You might want to delete it and run `npm install` again to re-create it